benchmark_analyzer/src/model/context/mod.rs

```diff
@@ -0,0 +1,38 @@
+//!
+//! A context for benchmarking, passed by compiler-tester.
+//!
+
+///
+/// A context for benchmarking, passed by compiler-tester.
+///
+#[derive(Clone, Debug, Default, Eq, PartialEq, serde::Serialize, serde::Deserialize)]
+pub struct Context {
+    /// Unique identifier of the machine.
+    pub machine: String,
+    /// Target, for example as "eravm" or "evm".
+    pub target: String,
```

Comment:
Could we use `era_compiler_common::Target` here?

Reply:
Probably so. @antonbaliasnikov is "target" for LNT purposes always going to be `eravm` or `evm`? [Source of `era_compiler_common::Target`](https://github.com/matter-labs/era-compiler-common/blob/main/era-compiler-common/src/target.rs)

Reply:
Yes, we should be using those conventional names for targets. They are also used in LLVM.